qutebrowser/javascript/greasemonkey_wrapper.js

```diff
@@ -145,59 +145,57 @@
         }
     };
 
+    const unsafeWindow = window;
```

Comment:
hmm, was this always broken on webkit?

Reply:
I think so. Even if `window` and `unsafeWindow` are the same thing due to lack of support for `Proxy`, some scripts will break if they try to access `unsafeWindow` and it isn't defined.